azure/petctl.py

```diff
@@ -0,0 +1,348 @@
+from util import *
```

Comment:
Please avoid `import *` to be compatible with PEP 8: https://www.python.org/dev/peps/pep-0008/

Reply:
Ok, thanks!